api/namespace/application/routes.py

```diff
@@ -0,0 +1,76 @@
+from database.store import APPLICATIONS
+from api.namespace.application.models.application import application_inbound
+from api.namespace.application.models.application import application_full
+from api.namespace.application.models.application import application_status
+from api.namespace.application.application_ns import application_ns
+from flask import abort
+from flask_restx import reqparse
+from flask_restx import Resource
+
+
+@application_ns.route("/", methods=["GET", "POST"])
+class ApplicationCreate(Resource):
+    """
+    Create a new application
+    """
+
+    @application_ns.doc("create_application")
+    @application_ns.expect(application_inbound)
```

Comment:
we may want to pass in a boolean to @expect here to verify the payload (according to RESTX docs: https://flask-restplus.readthedocs.io/en/stable/swagger.html#the-api-expect-decorator)